scripts/foundry/transmuter/SetupHarvester.s.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { console } from "forge-std/console.sol";
+import { IVaultManagerFunctions } from "borrow/interfaces/IVaultManager.sol";
+import { IERC721Metadata } from "oz/token/ERC721/extensions/IERC721Metadata.sol";
+import { Enum } from "safe/Safe.sol";
+import { MultiSend, Utils } from "../Utils.s.sol";
+import { BaseHarvester, YieldBearingParams } from "transmuter/helpers/BaseHarvester.sol";
+import { ISettersGuardian, ISettersGovernor } from "transmuter/interfaces/ITransmuter.sol";
+import { TrustedType } from "transmuter/transmuter/Storage.sol";
+import { MultiBlockHarvester } from "transmuter/helpers/MultiBlockHarvester.sol";
+import "../Constants.s.sol";
+
+contract SetupHarvester is Utils {
+    function run() external {
+        bytes memory transactions;
+        uint8 isDelegateCall = 0;
+        uint256 value = 0;
+
+        uint256 chainId = vm.envUint("CHAIN_ID");
+
+        /** TODO  complete */
+        address harvester = 0x0A10f87F55d89eb2a89c264ebE46C90785a10B77;
+        uint64 targetExposure = 0.35e9;
+        address asset = EUROC;
+        address yieldBearingAsset = XEVT;
+        bool isGenericHarvester = false;
+        /** END  complete */
+
+        // Add harvester to trusted
+        {
+            address transmuter = _chainToContract(chainId, ContractType.TransmuterAgEUR);
+            bytes memory data = abi.encodeWithSelector(ISettersGovernor.toggleTrusted.selector, harvester);
```

Comment:
you need to inverse the signature function. this signature is the one from the harvester and inversely for l.44